form-validation.js

```diff
@@ -35,7 +39,7 @@ function validateDesk() {
     return true;
   }
   email.style.borderColor = 'red';
-  message.innerText = 'remove ALL Uppercase characters please';
+  message.innerText = 'Emails cannot take CAPITAL Letters submit valid email';
```

Comment:
- [ ] The error message displayed is obscure from a user. Please ensure that the error message is displayed just a little above or below the submit button of the form.

**Your form:**
![image](https://user-images.githubusercontent.com/12745474/218288260-ea6e889e-1d97-41ba-b757-8aa1ba14d108.png)